pkg/collectors/devices/pmc.go

```diff
@@ -15,10 +15,11 @@ import (
 )
 
 type PMCInfo struct {
+	Timestamp               string `fetcherKey:"date"                    json:"timestamp" `
 	TimeSource              string `fetcherKey:"timeSource"              json:"timeSource"`
 	ClockAccuracy           string `fetcherKey:"clockAccuracy"           json:"clockAccuracy"`
 	OffsetScaledLogVariance string `fetcherKey:"offsetScaledLogVariance" json:"offsetScaledLogVariance"`
-	ClockClass              int    `fetcherKey:"clockClass"              json:"clockClass"`
+	ClockClass              int    `fetcherKey:"clockClass"              json:"clkcl"`
```

Comment:
Not sure if `clkcl` is meaningful enough

Reply:
Good point I've added a comment to the PR in `vse-sync-pp` we need to match there names for the JSON object handling to work.

Reply:
https://github.com/redhat-partner-solutions/vse-sync-pp/pull/28#discussion_r1285621099